doc/man/8/rbd.rst

```diff
@@ -333,8 +333,8 @@ Commands
   Enable the specified feature on the specified image. Multiple features can
   be specified.
 
-:command:`flatten` *image-spec*
-  If image is a clone, copy all shared blocks from the parent snapshot and
+:command:`flatten` [--encryption-format *encryption-format* --encryption-passphrase-file *passphrase-file*]... *image-spec*
```

Comment:
This line should stay is as in quincy -- there options doesn't exist there.

Reply:
I posted https://github.com/ceph/ceph/pull/56272.

Reply:
@idryomov @anthonyeleven I've approved Ilya's Quincy-only PR.